### ISSUE TITLE
Duda sobre el modulo de memoria

### ISSUE THREAD

**Issue by mSpadoni** [sisoputnfrba/foro#3903]
hola, en el modulo de memoria es necesario crear el archivo bitmap.dat como en la interfaz o esta bien si solo creo un array de bits y lo utilizo para indicar si un frame esta ocupado o no??

**Comment by f-and** [comment]
Buenas! No creo necesario un archivo bitmap dedicado ya que la memoria no es persistente entre sesiones, como una computadora real, si se apaga la misma entonces se limpia toda la memoria (exceptuando cosas como hibernar o "dormir" la PC). Con un bit_array ya seria suficiente.

**Comment by mSpadoni** [comment]
joya gracias, y tengo una duda con el acceso a tabla de paginas, tengo que devolverle a la cpu el marco asociado a la pagina que me manda??

**Comment by RaniAgus** [comment]
Exacto, dado pid y número de página debe devolver el número de marco para que la MMU pueda continuar con la traducción de dirección lógica a física

**Comment by mSpadoni** [comment]
joya 
muchas gracias
